features/support/env.rb

```diff
@@ -6,9 +6,30 @@ def run_command(cmd, timeout = nil)
     exec_cmd = cmd =~ /^rspec/ ? "bin/#{cmd}" : cmd
     # Ensure bundler env vars are unset
     unset_bundler_env_vars
-    # Ensure the correct Gemfile is found
+    # Ensure the correct Gemfile and binstubs are found
     in_current_directory do
-      super(exec_cmd, timeout)
+      with_unbundled_env do
```

Comment:
I _think_ this call to `with_unbundled_env` is not needed.

Reply:
Yep, #2244 did not need it.